System.IO.Packaging/FileFormatException.cs

```diff
@@ -8,12 +8,10 @@
 //
 //---------------------------------------------------------------------------
 
-using System;
-using System.Security;
-using System.IO.Packaging;
-
 namespace System.IO
```

Comment:
System.IO.FileFormatException was part of the original WindowsBase dll, and it was in the System.IO namespace.  It would be a breaking change to have it in a different namespace, and further, we do not want to require WindowsBase.dll just for the purposes of getting that exception - it causes name conflicts with System.IO.Packaging, and there is nothing else in that DLL that is commonly used.  The option I chose was to put it into System.IO.Packaging, and in its original namespace.